docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -90,23 +100,27 @@ CMD ["/bin/bash"]
 ```
 
 ### Executing Containerized Tests in OpenShift CI
+
 Using a container is very easy in OpenShift CI. There are several ways it can be done (from a registry, a Dockerfile, an image stream, _etc._). However, for the purposes of this guide, we will focus on using the Dockerfile.
 
 #### Define the Container
+
 To define a container in OpenShift CI, two stanzas need to be added to the configuration file in OpenShift CI. Keep in mind, the configuration files in OpenShift CI are named based on the organization, repository, and branch of the test repository. So when we specify the Dockerfile's location or the context of the build, it is relative to the root of the testing repository. 
 
 Pretend we have a test repository in the `CSPI-QE` organization named `mock_tests` and we'd like to use the `main` branch of the repository to test with. The configuration file in the `openshift/release` repository would be `ci-operator/config/cspi-qe/mock_tests/cspi-qe-mock_tests-main.yaml`. 
 
 Inside of this pretend repository there is a directory named `dockerfiles` containing the Dockerfile we made above. In the root of the repository in the test file, `test_mock_tests.py`.
 
 The following is an abbreviated version of the configuration file for our test repository. The most important stanza to look at for this guide are:
```

Comment:
```suggestion
The following is an abbreviated version of the configuration file for our test repository. The most important stanzas to look at for this guide are:
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)